### ISSUE TITLE
Focus Other Chrome Windows

### ISSUE THREAD

**Issue by jtanderson** [fouad/chrome-tablight#6]
If you have multiple Chrome windows (same or different workspace), it properly indexes those tabs but does not focus/switch to that window when you select a tab from it.

**Comment by fouad** [comment]
You're right! To-do: https://developer.chrome.com/extensions/windows#method-update
